Content.Shared/Backmen/Surgery/SharedSurgerySystem.Steps.cs

```diff
@@ -224,7 +216,7 @@ private void OnTendWoundsCheck(Entity<SurgeryTendWoundsEffectComponent> ent, ref
             || group.Any(damageType => damageable.Damage.DamageDict.TryGetValue(damageType, out var value)
                 && value > 0)
             || !TryComp(args.Part, out BodyPartComponent? bodyPart)
-            || bodyPart.Integrity < BodyPartComponent.MaxIntegrity)
+            || bodyPart.Integrity <= bodyPart.MaxIntegrity)
```

Comment:
For context, all of the On{stepname}Check functions are checking for a conditional, and if this conditional isnt passed, then we count the surgery as still not completed. Otherwise it is marked as such.

So in this case, if your integrity is at say, 100 dmg, the condition is not true and it will be marked as complete. Shouldnt it be > since we flipped the logic around?